packages/core/src/new-api/internal/module-builder.ts

```diff
@@ -363,6 +367,22 @@ export class IgnitionModuleBuilderImplementation<
     return future;
   }
 
+  public getParameter<ParamType extends SolidityParamType>(
```

Comment:
```suggestion
  public getParameter<ParamType extends SolidityParamType = any>(
```

I checked what we discussed earlier today, and if we add this it can lead to a nicer UX. Otherwise you'd have to cast `unknowns` into other types when using params without a default value.

Reply:
done!